assets/js/app/Panel.js

```diff
@@ -66,50 +49,107 @@ LocusZoom.Panel = function() {
     this.getBaseId = function(){
         return this.parent.id + "." + this.id;
     };
+
+    // Initialize the layout (should this happen in initialize()?)
+    this.initializeLayout();
     
     return this;
     
 };
 
+LocusZoom.Panel.prototype.initializeLayout = function(){
+
+    // Set panel dimensions, origin, and margin or fall back to default values
+    this.layout.width      = this.layout.width      || 0;
```

Comment:
Looking at the entire file now, I see that the default layout for a panel should have already been passed though and set in the constructor before this function is called. So my initial comments probably aren't actually relevant.